src/icons/fileIcons.ts

```diff
@@ -316,6 +315,9 @@ export const fileIcons: FileIcons = {
         'psql',
         'db',
         'db3',
+        'dblite',
+        'dblite3',
+        'debugsymbols',
```

Comment:
And maybe one final question, what is the debugsymbols about and how's it related to the database icon?

Reply:
This one is admittedly more niche, as the main place I've found it is the Godot repo. It's similar to `.pdb`, but constructed directly after the fact by extracting the debug data from the compiled executable & truncating it afterward. I would call it the Unix equivalent of `.pdb`, but with much less comparative widespread use I'm hesitant to do so; but, functionally, it can be viewed as such

Reply:
thanks for the explanation, that sounds good 👍